ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -980,6 +980,16 @@ tests:
         DISABLED_HOST_VALIDATIONS=belongs-to-majority-group
       PLATFORM: nutanix
     workflow: assisted-nutanix
+- as: e2e-ai-operator-ztp-hypershift-zero-nodes
```

Comment:
sure, moved.

Reply:
move it up so it stays close to other ztp jobs